astropy/modeling/fitting.py

```diff
@@ -532,7 +696,7 @@ def __call__(self, model, x, y, z=None, weights=None, rcond=None):
                 good = ~model_rhs.mask
                 model_lhs = lhs[good]
                 model_rhs = model_rhs[good][..., np.newaxis]
-
+                a = model_lhs
```

Comment:
I ran into some conflicts with this PR and #10687, and I do not understand how this line can work if there are multiple models. `a` should be different for each model isn't it ? cc @nden
Also `uncertanties` is used everywhere tough that should be `uncertainties`, right?

Reply:
I also spotted invertable -> invertible.

Reply:
ahhh whoops! i am way too reliant on my spellchecker. I will fix these typos and take a look at the issue involving multiple models.

Reply:
Thank you! It would be good to figure that out so we can merge the other PR. I've only just seen this and am not clear enough on what it's doing to offer any immediate insight.

Reply:
@cshanahan1 - I already opened a PR for the typos (#10764), and can add the invertable correction there (thanks @jehturner ).